js/client/app/initMember.js

```diff
@@ -0,0 +1,43 @@
+import { loadTeam } from "../../serverAPI/team/loadTeam.js";
+import { addMember } from "../../serverAPI/user/addMember.js";
+import { drawMember } from "./drawMember.js";
+
+export async function initMember() {
+  const teamID = location.hash.split("#")[1];
+  const members = await loadTeam(teamID);
+
+  if (members._id !== teamID) {
+    alert("operation not successful(team id mismatch).");
+  }
+
+  initHeader(members.name);
+
+  for (const member of members.members) {
+    drawMember(member);
+  }
```

Comment:
for문 대신 다양한 Javascript Array의 순회메서드를 사용해보시면 더 좋을거 같아요!

```suggestion
members.mebers.forEach((member) => drawMember(member)))
```